scripts/qemu.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, binary, args=[], wrapper=[], name=None, test_dir="/var/tmp",
         self._qemu_log_path = os.path.join(test_dir, name + ".log")
         self._popen = None
         self._binary = binary
-        self._args = list(args) # Force copy args in case we modify them
+        self.args = list(args) # Force copy args in case we modify them
```

Comment:
I believe `self._args` is in use in `tests/qemu-iotests/iotests.py`. So yes to this change but please adjust all the places it's being used in.